services/cirrus/cirrus.service.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.alternatives()
```

Comment:
Looks like the API response uses the key `status` instead of `message`
```suggestion
  status: Joi.alternatives()
```